tools/osx/crosstool/cc_toolchain_config.bzl.tpl

```diff
@@ -5402,6 +5391,88 @@ def _impl(ctx):
         ],
     )
 
+    headerpad_feature = feature(
+        name = "headerpad",
+        enabled = True,
+        flag_sets = [
+            flag_set(
+                actions = all_link_actions + [
+                    ACTION_NAMES.objc_executable,
+                    ACTION_NAMES.objcpp_executable,
+                ],
+                flag_groups = [flag_group(flags = ["-headerpad_max_install_names"])],
+                with_features = [with_feature_set(not_features = [
+                    "bitcode_embedded",
+                    "bitcode_embedded_markers",
+                ])],
+            ),
+        ],
+    )
+
+    if (ctx.attr.cpu == "ios_arm64"
+        or ctx.attr.cpu == "ios_arm64e"
+        or ctx.attr.cpu == "ios_armv7"
+        or ctx.attr.cpu == "tvos_arm64"
+        or ctx.attr.cpu == "watchos_arm64_32"
+        or ctx.attr.cpu == "watchos_armv7k"
+        or ctx.attr.cpu == "darwin_x86_64"):
```

Comment:
So basically all Mac apps?

Reply:
Does this list need to include just "darwin"?

Reply:
I don't think there's a toolchain for just "darwin" as a CPU, so this should be fine.

Reply:
Right, the `darwin` whitelisting was just for apple_platform_type, that should still translate to `darwin_x86_64` cpu

Reply:
re https://github.com/bazelbuild/bazel/pull/7151

Reply:
thanks

Reply:
To answer the other question, all mac apps that opt into this feature with `--apple_bitcode=embedded` will get this yes. But that's currently not on by default (there's a TODO stating that it should be)